resolutions/numbers.md

```diff
@@ -3,3 +3,5 @@
 [2020-01](approved/2020-01.md) - Formation of Charter Maintenance Working Group
 
 [2020-02](proposed/2020-02.md) - Formation of Outreach Working Group
+
+[2020-03](proposed/2020-03.md) - Suggested changes for Steering Committee member selection
```

Comment:
This file wasn't updated to capture resolutions 3 and 4, at least one of which was mine (Mea Culpa). This one should be resolution 5.

Resolutions 3 and 4 are in the approved folder.

Reply:
All fixed. :)